pkg/actions/actions.go

```diff
@@ -325,7 +324,7 @@ func (o *optionLoader) LoadClientConfig() *client.Config {
 func (o *optionLoader) LoadApp() app.App {
 	i := o.load(OptionApp)
 	if i == nil {
-		o.err = ErrNotInApp
+		o.err = errors.Wrap(ErrInvalidApp, "found app.yaml but cannot load app")
```

Comment:
The other way around - you'd need to wrap `o.err` with your context, so assuming have extra context that is useful it might look something like:

```
o.err = errors.Wrap(o.err, "extra context about what you were doing")
```

In this case it might be just as well to leave the original error be.

Reply:
Switched back to original error